templates/common/cloud-platform-alt-dns/files/coredns.yaml

```diff
@@ -13,6 +13,7 @@ contents:
         app: cloud-platform-infra-coredns
       annotations:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+        openshift.io/required-scc: privileged
```

Comment:
Same as above, reverts old @cybertron's change

Reply:
See https://github.com/openshift/machine-config-operator/pull/4830#discussion_r1991518715

Reply:
ACK even though it's now confusing. It means that our Pod requires privileged SCC but is not running as privileged itself. I hope we could understand it without PhD from auth, but not sure it's possible.

In any case, OK for me, sorry for the confusion